src/Algebra/Action/Structures/Raw.agda

```diff
@@ -0,0 +1,77 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Raw Actions of one (pre-)Setoid on another
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+open import Relation.Binary.Core using (Rel)
+
+module Algebra.Action.Structures.Raw
+  {c a ℓ r} {M : Set c} {A : Set a} (_≈ᴹ_ : Rel M ℓ) (_≈_ : Rel A r)
+  where
+
+open import Data.List.Base
+  using (List; []; _∷_; _++_; foldl; foldr)
+open import Data.List.Relation.Binary.Pointwise as Pointwise
+  using (Pointwise; []; _∷_)
+open import Function.Base using (id)
+open import Level using (Level; _⊔_)
+
+private
+  variable
+    x y z : A
+    m n p : M
+    ms ns ps : List M
+
+------------------------------------------------------------------------
+-- Basic definitions: fix notation
+
+record IsRawLeftAction : Set (a ⊔ r ⊔ c ⊔ ℓ) where
+  infixr 5 _ᴹ∙ᴬ_ _ᴹ⋆ᴬ_
```

Comment:
I can see the point of these pre- and post- superscripts, but as you yourself rename these away almost everywhere, maybe that indicates that they're not great names after all?

Reply:
Could be: an aspect of the evolution of the design I suppose, but at some point I had wanted a non-symmetric symbol to indicate the 'handedness' of the action, even if subsequently it is indeed a pain to write it. But using bare `_∙_` seemed to invite a horrible notational clash with that symbol coming  from `Monoid`... cf [your comment](https://github.com/agda/agda-stdlib/pull/2350/files#r1561953300) above on `Bundles`...